app/views/suppliers/show.html.erb

```diff
@@ -1,3 +1,14 @@
 <div id="supplier_home">
-  <h1><%= @supplier.name %> Supplier Page</h1>
+	<h1><%= @supplier.name %> Supplier Page</h1>
+	<ul>
+		<% @listings.each do |listing|  %>	
+			<li>
+				<% item = Item.find(listing.item_id)  %>
```

Comment:
I think we should merge it since all tests are passing and then we can focus on this during refactoring.